jms/src/main/scala/akka/stream/alpakka/jms/JmsConnector.scala

```diff
@@ -20,26 +23,18 @@ private[jms] trait JmsConnector { this: GraphStageLogic =>
 
   implicit protected var ec: ExecutionContext = _
 
-  protected var jmsConnection: Option[Connection] = None
+  protected var jmsConnection: Option[jms.Connection] = None
 
   protected var jmsSessions = Seq.empty[JmsSession]
 
   protected def jmsSettings: JmsSettings
 
-  protected def onSessionOpened(jmsSession: JmsSession): Unit = {}
-
   protected def fail: AsyncCallback[Throwable] = getAsyncCallback[Throwable](e => failStage(e))
 
-  private def onConnection = getAsyncCallback[Connection] { c =>
+  private def onConnection = getAsyncCallback[jms.Connection] { c =>
     jmsConnection = Some(c)
   }
 
-  private def onSession =
-    getAsyncCallback[JmsSession] { session =>
-      jmsSessions :+= session
-      onSessionOpened(session)
-    }
```

Comment:
Moved to `SourceStageLogic` in `JmsConsumerStage.scala` for this function only applies to consumers